src/openzaak/components/zaken/api/serializers/zaken.py

```diff
@@ -161,10 +162,12 @@ class Meta:
         fields = ("bronorganisatie", "startdatum")
 
     def create(self, validated_data):
-        return self.Meta.model.objects.generate(
-            validated_data["bronorganisatie"],
-            validated_data["startdatum"],
+        func = import_string(
+            settings.ZAAK_IDENTIFICATIE_GENERATOR_OPTIONS[
+                settings.ZAAK_IDENTIFICATIE_GENERATOR
+            ]
         )
+        return func(validated_data)
```

Comment:
It seems that checks are not run when running with uWSGI in production (https://docs.djangoproject.com/en/4.2/topics/checks/), so I think we should make sure that this at least always falls back on `openzaak.components.zaken.api.utils.generate_zaak_identificatie_with_start_datum_year`. Could you also add a test where `ZAAK_IDENTIFICATIE_GENERATOR` is set to a some value that isn't valid?

```suggestion
        func = import_string(
            settings.ZAAK_IDENTIFICATIE_GENERATOR_OPTIONS.get(
                settings.ZAAK_IDENTIFICATIE_GENERATOR
            , "use-start-datum-year")
        )
        return func(validated_data)
```